public/js/icinga/behavior/modal.js

```diff
@@ -90,6 +93,8 @@
         var _this = event.data.self;
         var $form = $(event.currentTarget).closest('form');
         var $modal = $form.closest('#modal');
+        // otherwise the form is submitted several times by clicking the "Submit" button several times
+        $form.find('input[type=submit]').prop('disabled', true);
```

Comment:
Please disable all form inputs like [here](https://github.com/Icinga/icingaweb2/blob/master/public/js/icinga/loader.js#L133). It may not just be the submit button that is submitting the form.